dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -391,45 +392,31 @@ private static ArtifactResults generateArtifactReport(
   }
 
   /**
-   * Returns a table where rows are jar files in {@code jarFilter}, columns are {@link
-   * SymbolProblem}, and values are class names referencing the problems. When {@code jarFilter} is
-   * {@code null}, jar files are not filtered.
+   * Returns a nested map where first keys are JAR files with symbol problems, second keys are the
```

Comment:
How about

`Partitions {@code symbolProblems} by the JAR file that contains the {@link ClassFile}.`